dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -114,10 +116,16 @@ public QuayImageRegistry(HttpClient client, ObjectMapper objectMapper, Token qua
             if (asString.isPresent()) {
                 RepoList repos;
                 try {
+                    // interesting, this relies upon our container object having the same fields
+                    // as quay.io's repositories
                     repos = objectMapper.readValue(asString.get(), RepoList.class);
                     LOG.info("RESOURCE CALL: " + url);
 
                     List<Container> containers = repos.getRepositories();
+                    // tag all of these with where they came from
+                    containers.stream().forEach(container -> container.setRegistry(Registry.QUAY_IO.name()));
+                    // not quite correct, they could be mixed but how can we tell from quay?
+                    containers.stream().forEach(container -> container.setMode(ContainerMode.AUTO_DETECT_QUAY_TAGS_AUTOMATED_BUILDS));
```

Comment:
I set this value after I updated all the tags.
Set to mixed if there is at least one tag without automation, otherwise its the automated mode.